dbusmock/__main__.py

```diff
@@ -24,46 +24,54 @@
 
 
 def parse_args():
-    '''Parse command line arguments'''
-
-    parser = argparse.ArgumentParser(description='mock D-Bus object')
-    parser.add_argument('-s', '--system', action='store_true',
-                        help="put object(s) on system bus (default: session bus or template's SYSTEM_BUS flag)")
-    parser.add_argument('--session', action='store_true',
-                        help="put object(s) on session bus (default without template; overrides template's SYSTEM_BUS flag)")
-    parser.add_argument('-l', '--logfile', metavar='PATH',
-                        help='path of log file')
-    parser.add_argument('-t', '--template', metavar='NAME',
-                        help='template to load (instead of specifying name, path, interface)')
-    parser.add_argument('name', metavar='NAME', nargs='?',
-                        help='D-Bus name to claim (e. g. "com.example.MyService") (if not using -t)')
-    parser.add_argument('path', metavar='PATH', nargs='?',
-                        help='D-Bus object path for initial/main object (if not using -t)')
-    parser.add_argument('interface', metavar='INTERFACE', nargs='?',
-                        help='main D-Bus interface name for initial object (if not using -t)')
-    parser.add_argument('-m', '--is-object-manager', action='store_true',
-                        help='automatically implement the org.freedesktop.DBus.ObjectManager interface')
-    parser.add_argument('-p', '--parameters',
-                        help='JSON dictionary of parameters to pass to the template')
-    parser.add_argument('-e', '--exec', nargs=argparse.REMAINDER,
-                        help='Command to run in the mock environment')
+    """Parse command line arguments"""
+
+    parser = argparse.ArgumentParser(description="mock D-Bus object")
+    parser.add_argument(
+        "-s",
+        "--system",
+        action="store_true",
+        help="put object(s) on system bus (default: session bus or template's SYSTEM_BUS flag)",
+    )
+    parser.add_argument(
+        "--session",
+        action="store_true",
+        help="put object(s) on session bus (default without template; overrides template's SYSTEM_BUS flag)",
+    )
+    parser.add_argument("-l", "--logfile", metavar="PATH", help="path of log file")
+    parser.add_argument("-t", "--template", metavar="NAME", help="template to load (instead of specifying name, path, interface)")
+    parser.add_argument(
+        "name", metavar="NAME", nargs="?", help='D-Bus name to claim (e. g. "com.example.MyService") (if not using -t)'
+    )
```

Comment:
This kind of formatting inconsistency makes me weep. Three different styles to format the same kind of function call.. Especially the third one makes no sense at all IMHO. Either everything into one line, or split into one arg per line. I suppose this needs some manual reformatting and arguing until human sense of beauty/readability and black eventually agree?

Reply:
unfortunately black isn't smart enough to know they're the same calls so it just goes by whether stuff fits on a line. In the first case the three args don't fit on one line within the line break which is why it's broken up in three different ways.

Looks like the way around this (sort-of) is to add a stray `,` after the help text and then each arg will be on one line. That blows out the code a bit though. Alternatively: `# fmt: off` works too for specific cases where we really don't want black to touch it.

Reply:
The change to 118 char lines actually helped to make this more consistent.

Reply:
Meh, not important enough. Let's fix pylint and be done with it.

Reply:
there's basically zero wriggle room, only whatever `black --help` suggests which is... line length and string quotes (apparently? that must be new-ish).